styleguide/components/00-settings/_colors.scss

```diff
@@ -0,0 +1,196 @@
+// Colors.
+$colors: (
+  'color-cyan': #009de0,
```

Comment:
remove color from this values :) Calling color('color-blue') is a bid stupid :D

Reply:
Agreed.